core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1279,6 +1279,7 @@ class KafkaApis(val requestChannel: RequestChannel,
       // get metadata (and create the topic if necessary)
       val (partition, topicMetadata) = CoordinatorType.forId(findCoordinatorRequest.data.keyType) match {
         case CoordinatorType.GROUP =>
+          authorizeClusterOperation(request, CLUSTER_ACTION)
```

Comment:
Interesting pragmatic check. I assume it would be beneficial for transactions as well, as they also depends on the (successful) creation of an internal topic?

Reply:
Hi, thanks for reviewing this. Indeed yes, it would be beneficial. I've just moved the authorizedOp on top